step-5/README.md

```diff
@@ -124,33 +135,36 @@ Create the function in your `methods: {..}` object, like this:
   },
 },
 ```
-Notice that the code above is almost an identical copy of the code from your `script.js` file, *_except_* that any mention of a formerly-global variable has been update to be prefixed with `this.`. That's how the Vue app knows that the variables in question are scoped to _*this*_ component.
+
+Notice that the code above is almost an identical copy of the code from your `script.js` file, _*except*_ that any mention of a formerly-global variable has been update to be prefixed with `this.`. That's how the Vue app knows that the variables in question are scoped to _*this*_ component.
 
 You may also see a syntax error under the `matchCards()` function call - that's because you have not yet defined that function!
 
 Before you get to the `matchCards()` function, you will want to add the `flipCard` function as a click event callback. With Vue, this is easy...
 
 In your `GameBoard.vue` component, find your `<li>` tag:
+
 ```html
-<li v-for="(cardInfo, index) in cardsData" :key="index" class="card">
+<li v-for="(cardInfo, index) in cardsData" :key="index" class="card"></li>
```

Comment:
I wonder if your code editor has some setting that added this closing tag? I don't think this closing tag should be shown in the example here, please remove this.

Reply:
Yes, it was the vs code automatically including the closing tag. Removed the </li>